### PR TITLE
cart page

### PR DESCRIPTION
**Pull request by tulasikurete**

### CODE REVIEW

File: src/app/components/Cart.tsx [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+                  >
+                    Done
+                  </Button>
+                )}
+                {current > 0 && (
+                  <Button style={{ margin: "0 8px" }} onClick={() => prev()}>
+                    Previous
+                  </Button>
+                )}
+              </div> */}
```
Comment (NagaDurga16) [inline src/app/components/Cart.tsx]: please remove all comments

**Comment by tulasikurete** [comment]
image

**Comment by tulasikurete** [comment]
image

**Review by NagaDurga16 — APPROVED** [review]

**Comment by tulasikurete** [comment]
image

**Review by saiganesharaja — APPROVED** [review]

**Review by arajajyothibabu — APPROVED** [review]

**Pull request merged**